### PR TITLE
[risk=no][RW-10815] Repair all instances of .npmrc usage and update some Readme files

### PR DESCRIPTION
**Pull request by dmohs**
Narrows all usage of the Fontawesome license to use the key only rather than downloading a whole .npmrc file.

Not testing locally since CircleCI was primary failure mode.

Readme files updated and merged where appropriate.

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [x] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [x] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: ui/libproject/devstart.rb [hunk 1]
```diff
@@ -33,7 +33,8 @@ def build(cmd_name, args)
 
   common = Common.new
 
-  common.run_inline %W{gsutil cp gs://all-of-us-workbench-test-credentials/.npmrc ..}
+  fontawesomeCredsLineFilePath = "gs://all-of-us-workbench-test-credentials/dot-npmrc-fontawesome-creds-line.txt"
+  common.run_inline "gsutil cp #{fontawesomeCredsLinePath} >> ~/.npmrc"
```
Comment (jmthibault79) [inline ui/libproject/devstart.rb:37]:
```suggestion
  common.run_inline "gsutil cp #{fontawesomeCredsLineFilePath} >> ~/.npmrc"
```

File: ui/libproject/devstart.rb [hunk 2]
```diff
@@ -33,7 +33,8 @@ def build(cmd_name, args)
 
   common = Common.new
 
-  common.run_inline %W{gsutil cp gs://all-of-us-workbench-test-credentials/.npmrc ..}
+  fontawesomeCredsLineFilePath = "gs://all-of-us-workbench-test-credentials/dot-npmrc-fontawesome-creds-line.txt"
```
Comment (jmthibault79) [inline ui/libproject/devstart.rb:36]: works

File: .circleci/config.yml [hunk 3]
```diff
@@ -282,9 +282,12 @@ commands:
           key: v5-ui-cache-{{ .Branch }}-{{ checksum "~/workbench/ui/yarn.lock" }}
       - gcloud-auth-login
       - run:
-          name: "Download .npmrc file with Fontawesome Pro license key so FA icon SVG packages can be installed"
+          name: "Download Fontawesome Pro license key"
```
Comment (jmthibault79) [inline .circleci/config.yml:285]: observed to work correctly in CircleCI

File: README.md [hunk 4]
```diff
@@ -103,48 +103,7 @@ The `status` column enum values can be found in `org.pmiops.workbench.db.model.S
...
-```
-
-Before launching or testing the UI, yarn must first install the neccessary packages. From the `ui/` directory:
-```Shell
-yarn install
-```
-
-To launch the local UI:
-```Shell
-yarn dev-up-local
```
[the remark this answers is not in the document]
Comment (dmohs) [reply README.md:118]: I worry about maintaining this list. What do you think about [this](https://github.com/all-of-us/workbench/pull/7933/commits/98bebc3d97214c33d13bd31c22fc37a7de8b7e40) as an alternative?

**Pull request merged**